packages/cursorless-vscode/src/ide/vscode/hats/VscodeHatRenderer.ts

```diff
@@ -218,7 +229,16 @@ export default class VscodeHatRenderer {
     this.decorationMap = Object.fromEntries(
       Object.entries(this.enabledHatStyles.hatStyleMap).map(
         ([styleName, { color, shape }]) => {
-          const { svg, svgWidthPx, svgHeightPx } = hatSvgMap[shape];
+          const svgInfo = hatSvgMap[shape];
+
+          if (svgInfo == null) {
+            return [
+              styleName,
+              vscode.window.createTextEditorDecorationType({}),
+            ];
+          }
```

Comment:
We're now much more robust if we don't like one of the svgs. Before we would fail with no error message and all hats would stop working if there was a missing viewBox. Now we show an error message and the broken hat disappears